volume/struct.go

```diff
@@ -50,6 +50,13 @@ const (
 	DistDispStripe
 )
 
+// Volume list
+// Represent the list of volume during GetVol request
+type VolumeList struct {
```

Comment:
Like I mentioned in my previous comment, there is no need for a separate type for this. A map is good enough.